src/main/java/com/example/javaOjt/repositories/daos/AuthorDao.java

```diff
@@ -1,9 +1,13 @@
 package com.example.javaOjt.repositories.daos;
 
-import java.util.List;
 import com.example.javaOjt.beans.dtos.AuthorWithBookDTO;
+import com.example.javaOjt.beans.entities.Author;
+import java.util.List;
 
 public interface AuthorDao {
 
   List<AuthorWithBookDTO> getAuthorWithBookDTOsById(Integer id);
+
+  List<Author> authorsList(String attribute, String order);
```

Comment:
せっかく enum 使っているので使いましょう！String より Enum の方は方安全なので。

```suggestion
  List<Author> authorsList(AuthorSortBy attribute, Order order);
```